administrator/components/com_admin/script.php

```diff
@@ -2045,6 +2045,7 @@ public function deleteUnexistingFiles($dryRun = false, $suppressOutput = false)
             '/media/com_contenthistory/js/admin-history-versions.min.js.gz',
             // From 5.0.0-beta1 to 5.0.0-beta2
             '/libraries/src/Cache/Storage/WincacheStorage.php',
+            '/language/en-GB/lib_simplepie.sys.ini',
```

Comment:
```suggestion
            '/language/en-GB/lib_simplepie.sys.ini',
            '/libraries/src/Cache/Storage/WincacheStorage.php',
```
Alpha order files in one section, here the "// From 5.0.0-beta1 to 5.0.0-beta2" secition. Hint: Folders (not the case here) would be sorted reverse, o.e. descending.